www/main/viewer/ShabadDeck/ShabadDeck.jsx

```diff
@@ -145,6 +145,9 @@ function ShabadDeck() {
     }
     if (ceremonyId && isCeremonyBani) {
       loadCeremony(ceremonyId).then(ceremonyVerses => {
+				if (ceremonyVerses.flat) {
```

Comment:
Veerji same here, the if will always be true. 

Also, if I'm not wrong this `flat` is an array method. Do you think this will give a result without calling it like `flat()`. Please help me remove this confusion veerji.

Reply:
same as above